src/util/colorspace_oklab.cpp

```diff
@@ -317,119 +91,18 @@ ColorOKLCh::ColorOKLCh(Color& c) :
   ColorOKLab lab = linear_srgb_to_oklab(rgb);
   *this = lab_to_lch(lab);
   if (C < 0.00001f)
-    // Deterministic behaviour when increasing chroma of greyscale colours.
+    // Deterministic behaviour for greyscale colors
     h = 0.0f;
 }
 
-Color
-ColorOKLCh::to_srgb() const
-{
-  ColorOKLCh c = *this;
-  ColorOKLab lab = lch_to_lab(c);
-  ColorRGB rgb = oklab_to_linear_srgb(lab);
-  if (!rgb.is_valid()) {
-    c.clip_chroma();
-    // Gamut clipping; reduce chroma when needed.
-    lab = lch_to_lab(c);
-    rgb = oklab_to_linear_srgb(lab);
-  }
-  if (!(rgb.r > -0.001f && rgb.r < 1.001f && rgb.g > -0.001f && rgb.g < 1.001f
-      && rgb.b > -0.001f && rgb.b < 1.001f)) {
-    log_warning << "Colour out of bounds (after clipping): (" << rgb.r <<
-      ", " << rgb.g << ", " << rgb.b << ")" << std::endl;
-  }
-  return linear_srgb_to_srgb(rgb);
-}
-
 float
-ColorOKLCh::get_maximum_chroma() const
+ColorOKLCh::get_lr() const
```

Comment:
Is there a way to make the name of this function more descriptive?

Reply:
Yes, I have renamed it to get_modified_lightness now but I'm not sure if it's a good name.
It applies the lightness modification from https://bottosson.github.io/posts/colorpicker/#intermission---a-new-lightness-estimate-for-oklab.